### ISSUE TITLE
Incorrect Links under API Reference 

### ISSUE THREAD

**Issue by robotjellyzone** [tensorflow/tensorflow#52253]
## URL(s) with the issue:

Please provide a link to the documentation entry : 
https://www.tensorflow.org/community/contribute/docs#api_reference

### Clear description

Under the description of  "API Reference", there is a link to a google doc for [Tensorflow 2 API Docs Advice](https://docs.google.com/document/d/1e20k9CuaZ_-hp25-sSd8E8qldxKPKQR-SkwojYr_r-U/preview#). This doc is having some links which are not working and may create confusion for any beginner who wants to start contributing to docs. 
Links like - a. link of [tensorflow docs task tracker](https://docs.google.com/spreadsheets/d/1p3vqbocbKmcZQGrlxk9m2jHuleu8yr-XRbfKum0IDD8/edit?usp=sharing) - this doc is not currently under consideration so this (along with its related info) should be removed to avoid any confusion. 
b.  [docstring links](https://www.tensorflow.org/community/documentation) - this link is not working (needs to be correctly linked)
c.  links given for [examples , guide & tutorials](https://github.com/tensorflow/docs/tree/master/site/en/r2) - incorrect link
d. link for [python API documentation](https://www.tensorflow.org/community/documentation#generating_python_api_documentation) - incorrect link

So, overall from a beginner's point of view, I can say that this FAQ doc for API is not much useful & helpful So I'll suggest either update it or remove it!

**Comment by robotjellyzone** [comment]
I can update this doc if this issue seems to be a correct doc-bug and I will surely make one PR in no time! Just need an approval ! @lamberta

**Comment by lamberta** [comment]
Thanks @robotjellyzone , added you to the assignees

**Comment by tilakrayal** [comment]
@robotjellyzone ,
Please feel free to submit a PR for the requested change or share the link where requested change is to be made.Thanks!

**Comment by bhack** [comment]
Yes what I meant generally (it isn't only for the Docs repository) is to find a balance between the website infos and something technical that could let you to quickly contribute a PR just starting from the community file on the repo and without going to duplicate the source of truth or request to go forth and back too much from Github to the website.

**Comment by robotjellyzone** [comment]
Ok Got it! I try to do it first according to my understanding and then will wait for your suggestions of improvements if any!

**Comment by tilakrayal** [comment]
The issue will move to closed status once the PR is merged.

**Comment by robotjellyzone** [comment]
Hi @lamberta 👋🏻 ! I have added my pr at [#1954](https://github.com/tensorflow/docs/pull/1954) so can you please review that PR & provide me with improvement pointers if any !

**Comment by robotjellyzone** [comment]
Hi, @tilakrayal  the #[pr1954](https://github.com/tensorflow/docs/pull/1954) for this issue is already merged so can you please close this issue by linking that pr to this issue !

**Comment by robotjellyzone** [comment]
Ok no problem ! I am closing this issue . Pr - [#1954](https://github.com/tensorflow/docs/pull/1954)
